spec/roman_numerals_spec.rb

```diff
@@ -0,0 +1,97 @@
+require 'roman_numerals'
+
+RSpec.describe RomanNumerals do
+  roman_numeral_converter = RomanNumerals.new
```

Comment:
Use let or before here. This works in this case because there's no state in your RomanNumerals class, but you'll usually want to create a new instance for every test. This will create one instance when the class is evaluated, and that same instance is used in all of your tests.